### PR TITLE
Display user avatar and details in navigation drawer.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
@passy @pautomas

### CODE REVIEW

File: app/src/main/java/com/twitter/tweetducker/model/User.java [hunk 1]
```diff
@@ -3,16 +3,21 @@
...
     public long id;
     public String name;
-    public String screenname;
+    public String screenName;
     public String description;
     public String profileImageUrlHttps;
 
+    public String getAvatarUrl() {
+        // Be better than this... :(
+        return profileImageUrlHttps.replace("_normal.", "_400x400.");
```
Comment (passy) [inline app/src/main/java/com/twitter/tweetducker/model/User.java]: :(

File: app/src/main/res/layout/nav_header_main.xml [hunk 2]
```diff
@@ -12,23 +12,24 @@
...
         android:paddingTop="@dimen/nav_header_vertical_spacing"
         android:src="@drawable/ic_tweetducker" />
 
     <TextView
+        android:id="@+id/screen_name"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:paddingTop="@dimen/nav_header_vertical_spacing"
-        android:text="Android Studio"
+        android:text="\@screen_name"
```
Comment (passy) [inline app/src/main/res/layout/nav_header_main.xml]: If you use `tools:text` here, you see the text in the preview but it won't accidentally show in the prod app.

**Comment by passy** [comment]
#shipit

File: app/src/main/res/layout/nav_header_main.xml (hunk 2 above)
Comment (daithiocrualaoich) [inline app/src/main/res/layout/nav_header_main.xml]: Done.

File: app/src/main/java/com/twitter/tweetducker/model/User.java (hunk 1 above)
Comment (daithiocrualaoich) [inline app/src/main/java/com/twitter/tweetducker/model/User.java]: Me too, sorry. :(

**Pull request merged**